test/integration/daemonset/daemonset_test.go

```diff
@@ -498,6 +533,50 @@ func TestSimpleDaemonSetLaunchesPods(t *testing.T) {
 	})
 }
 
+func TestSimpleDaemonSetRestartsPods(t *testing.T) {
+	for _, podPhase := range []v1.PodPhase{v1.PodSucceeded, v1.PodFailed} {
+		t.Run(fmt.Sprintf("%s_%v", t.Name(), podPhase), func(tt *testing.T) {
```

Comment:
`t.Name()` is already added by the test framework

```suggestion
		t.Run(podPhase, func(tt *testing.T) {
```